.github/workflows/backport-prs.yml

```diff
@@ -0,0 +1,74 @@
+name: Backport PRs
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  Backport:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged && contains( github.event.pull_request.labels.*.name, 'backport' )
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 100
+
+      - name: Set up git
+        run: |
+          git config --global user.email github-actions[bot]@users.noreply.github.com
+          git config --global user.name github-actions[bot]
+
+      - name: Get backport branch
+        id: get-branch
+        run: |
+          git fetch --no-tags origin 'refs/heads/*.x:refs/remotes/origin/*.x'
+          echo "::set-output name=backport-branch::$(git branch -r | grep '.*.x' | sort | tail -n 1 | cut -d / -f 2)"
```

Comment:
Will this break on double-digit minor versions? eg will
```
1.1.x
1.5.x
1.10.x
```

sort to
```
1.1.x
1.10.x
1.5.x
```
and pull out the wrong recent branch? Looks like `sort -V` would get around this just fine.

Reply:
Good catch. I don't anticipate that being a real problem in practice--we're not keeping around these branches long term, so there shouldn't actually be more than 1 at a time. However, since I'm trying to be robust here, might as well actually be correct.

Reply:
Fixed.